libfwupd/fwupd.map

```diff
@@ -272,3 +272,10 @@ LIBFWUPD_1.1.2 {
     fwupd_device_to_variant_full;
   local: *;
 } LIBFWUPD_1.1.1;
+
+LIBFWUPD_1.1.3 {
+  global:
+    fwupd_device_get_estimated_flash_time;
```

Comment:
I'm not sure I like the name :( we've not used the "flash" verb yet -- and it's a bit long. Maybe alternates are `install_duration`, `approx_duration`, `estimated_duration`, `time_estimate`?

Reply:
Adjusted